core/arch/arm/kernel/boot.c

```diff
@@ -1292,6 +1292,17 @@ static void init_primary(unsigned long pageable_part, unsigned long nsec_entry)
 	init_sec_mon(nsec_entry);
 }
 
+#if defined(ARM32) && (CFG_TEE_CORE_LOG_LEVEL >= TRACE_DEBUG)
+static bool cpu_nmfi_enabled(void)
+{
+	uint32_t sctlr = 0;
```

Comment:
``` C
if (IS_ENALBED(ARM32))
    return read_sctlr() & SCTLR_NMFI;
return false;
```

Reply:
OK

Reply:
I backtrack a bit in here as `SCTLR_NMFI` is not defined for `ARM64`.